### PR TITLE
Lag and API optimization

### PR DESCRIPTION
**Pull request by nzapponi**
This is to reduce lag and number of API calls.
Requires some testing

### CODE REVIEW

**Comment by nzapponi** [comment]
FYI @nikonratm this is my take on #61 . What do you think? I'll test over the next few days.

File: src/accessories/entrySensor.js [hunk 1]
```diff
@@ -74,20 +74,25 @@ class SS3EntrySensor {
...
     }
 
-    async getState(callback) {
-        try {
-            let sensor = await this.getSensorInformation();
-
-            if (!sensor.status) {
-                throw new Error('Sensor response not understood');
+    async getState(callback, forceRefresh = false) {
+        if (forceRefresh) {
```
Comment (nikonratm) [inline src/accessories/entrySensor.js]:
```suggestion
        if (!forceRefresh) {
```

**Comment by nikonratm** [comment]
This looks great, what I imagined after our last conversation about only pinging SS if needed. Will start testing later today. Caught 1 error, I think.

**Comment by nzapponi** [comment]
Good catch, thanks, fixed.

**Comment by nzapponi** [comment]
Merged into master! @nikonratm let's test for a couple of days before releasing on npm.

**Comment by nikonratm** [comment]
In testing so far this has been great, more responsive! Cheers

**Pull request merged**